roles/swift-ring/tasks/main.yml

```diff
@@ -13,3 +13,4 @@
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
            bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
   when: not should_boostrap|success
+  notify: restart swift services
```

Comment:
Correct -swift services automatically reads the rings once started. However, they need to be there before services can start. Current ansible scripts try to start service once conf file for that service is written (like other openstack services). Trying to rearrange roles, in my opinion, could lead to more convuled way of doing things/missed dependencies since some swift functionality/tools may not have been installed yet. I will remove the "notify: restart.. " block from Swift-ring role.